client/src/app/tabs/bpmn-shared/util/__tests__/configureSpec.js

```diff
@@ -159,11 +238,48 @@ describe('tabs/bpmn/util - configure', function() {
         expect(warnings).to.have.length(2);
 
         expect(warnings[0].message).to.eql(
-          'bpmn.modeler.moddleExtension is missing <name> property'
+          'A bpmn moddle extension plugin is missing a <name> property'
         );
 
         expect(warnings[1].message).to.eql(
-          'bpmn.modeler.moddleExtension overrides moddle extension with name <existing>'
+          'A bpmn moddle extension with name <existing> was overriden due to a clash'
+        );
+      });
+
+      it('bpmn.modeler.moddleExtension && bpmn.{platform}.modeler.moddleExtension', function() {
+
+        // given
+        var existingGenericExtension = { name: 'existingP' };
+        var existingGenericExtension2 = { name: 'existingC' };
+        var existingPlatformExtension = { name: 'existingP' };
+        var existingCloudExtension = { name: 'existingC' };
+
+        var getPlugins = setupPlugins([
+          [ 'bpmn.modeler.moddleExtension', existingGenericExtension ],
+          [ 'bpmn.modeler.moddleExtension', existingGenericExtension2 ],
+          [ 'bpmn.platform.modeler.moddleExtension', existingPlatformExtension ],
+          [ 'bpmn.cloud.modeler.moddleExtension', existingCloudExtension ]
+        ]);
+
+        // when
+        var {
+          warnings: warningsPlatform
+        } = configureModeler(getPlugins, undefined, undefined, 'platform');
```

Comment:
Yeah this is unfortunately the downside of JS not supporting named parameters, and almost every single parameter being optional, but I can also shift them around to avoid this, although I don't know if that's worth it.

Reply:
it's fine, as it's only for the tests. In the implementation, we always use all parameters.

Reply:
I'll just change these to null, it looks a little less irritating